picard/ui/itemviews/custom_columns/view_selector.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+#
+# Picard, the next-generation MusicBrainz tagger
+#
+# Copyright (C) 2025 The MusicBrainz Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
+
+"""View selector (for `add_to`) for custom columns."""
+
+from PyQt6 import (
+    QtCore,
+    QtWidgets,
+)
+
+from picard.i18n import gettext as _
+
+from picard.ui.itemviews.custom_columns.shared import (
+    get_ordered_view_presentations,
+)
```

Comment:
I think this single import might fit on a single line without exceeding the maximum line length.